tools/osv-linter/internal/pkgchecker/package_check.go

```diff
@@ -127,6 +127,15 @@ func existsInNuget(pkg string) bool {
 
 // Validate the existence of a package in Packagist.
 func existsInPackagist(pkg string) bool {
+	// most drupal packages are published in a dedicated repository, so check there first
```

Comment:
It looks like most comments in this file use sentence casing and punctuation so this should be `Most Drupal...first.` We also capitalize Drupal.

Reply:
I've used lowercase here as this is referring to the package namespace than the organisation